contracts/DeusBridge.sol

```diff
@@ -260,12 +231,12 @@ contract DeusBridge is IDeusBridge, Ownable, Pausable {
     /* ========== VIEWS ========== */
 
     // This function use pool feature to handle buyback and recollateralize on DEI minter pool
-    function collatDollarBalance(uint collat_usd_price) public view returns (uint) {
+    function getCollateralBalance(uint collat_usd_price) public view returns (uint) {
```

Comment:
The name of this function cannot be changed because this ABI has been used in other contacts.

Reply:
If we deploy our new contract this won't be an issue since we could just import the new ABI. Aside from that, I suggest we change this function name in those other instances as well for readability sake. After all, this *is* a getter function so it make sense to prepend it with `get`

Reply:
I agree with you, but unfortunately, it is used in the DEI.sol contract. so we can't change it easily. (+ we forked Frax.sol and it had the same name there). BTW you are right.